### PR TITLE
Add impl IntoIterator for MultiHandler to get access to inners

### PR DESCRIPTION
**Pull request by kpp**
To access the inners of MultiHandler. This PR is required by https://github.com/paritytech/substrate/issues/10370#issuecomment-981555744.

### CODE REVIEW

File: swarm/src/handler/multi.rs [hunk 1]
```diff
@@ -319,6 +319,16 @@ where
     }
 }
 
+/// Split `MultiHandler` into parts.
```
Comment (mxinden) [inline swarm/src/handler/multi.rs]:
```suggestion
/// Split [`MultiHandler`] into parts.
```

**Review by mxinden — COMMENTED** [review]
Looks good to me. Mind adding a changelog entry @kpp?

File: swarm/src/handler/multi.rs (hunk 1 above)
Comment (kpp) [reply swarm/src/handler/multi.rs]: Done

**Review by mxinden — APPROVED** [review]
Thanks @kpp. 

I hope to find time to do a release sometime this week.

**Pull request merged**